react/features/toolbox/components/native/OverflowMenu.js

```diff
@@ -56,23 +56,19 @@ class OverflowMenu extends Component<Props> {
      * @returns {ReactElement}
      */
     render() {
+        const buttonProps = {
+            afterClick: this._onCancel,
+            showLabel: true,
+            styles: overflowMenuItemStyles
+        };
+
         return (
             <BottomSheet onCancel = { this._onCancel }>
-                <AudioRouteButton
-                    showLabel = { true }
-                    styles = { overflowMenuItemStyles } />
-                <ToggleCameraButton
-                    showLabel = { true }
-                    styles = { overflowMenuItemStyles } />
-                <AudioOnlyButton
-                    showLabel = { true }
-                    styles = { overflowMenuItemStyles } />
-                <RoomLockButton
-                    showLabel = { true }
-                    styles = { overflowMenuItemStyles } />
-                <PictureInPictureButton
-                    showLabel = { true }
-                    styles = { overflowMenuItemStyles } />
+                <AudioRouteButton { ...buttonProps } />
```

Comment:
If these components are so standard that they have exactly the same props, shouldn't these props should be their default values? Also: where do you plan to reuse these buttons?

Reply:
No, because the buttons can go anywhere: the toolbar, the overflow menu, or whatever, the style is what defines how they look / behave, which depends on where you decide to place them.

Reply:
Okay, this seems to be a difference in our coding style, but I'm okay with it this way too.

I just add my 2 cents as a clarification: what I meant is that it's not currently used anywhere and we don't have any future plans to use it anywhere ATM. From my point of view, optimising something for a future use where that future use is not visible on the horizon yet is a waste of time and an unnecessary complication in the code (again, it's my point of view, not a fact). And that unnecessary additional complication may even make our job more difficult in future use cases - similar future use cases that we don't foresee yet. So that's a 1-to-1 for me, except the additional work if the (premature?) optimisation is done :)